android-test/src/androidTest/java/okhttp/android/test/OkHttpTest.kt

```diff
@@ -391,9 +392,20 @@ class OkHttpTest {
   }
 
   private fun enableTls() {
+    val trustManager = handshakeCertificates.trustManager
+    Platform.get().log("tm $trustManager", Platform.WARN)
+
+    val toList = trustManager.acceptedIssuers.toList().map {
+      it.subjectDN
+    }
+
+    Platform.get().log("tmIssuers $toList", Platform.WARN)
+
+    // TODO: Bug fixed with new client
```

Comment:
Working on the fix, be good to get in for this release.

Reusing a client via newBuilder and setting trustManager doesn't use the correct CertificateChainCleaner.

Reply:
Ahhhh, that explains it. Good find.